kivy/core/window/__init__.py

```diff
@@ -2017,9 +2051,37 @@ def on_dropfile(self, filename):
             (ios, android etc.)
 
         .. versionadded:: 1.2.0
+
+        .. versionchanged:: 2.1.0
+            Renamed from `on_dropfile` to `on_drop_file`.
         '''
+        if self.get_property_observers('on_dropfile'):
+            self.dispatch('on_dropfile', filename)
+
+    @deprecated(msg='Deprecated in 2.1.0, use on_drop_file event instead. '
+                    'Event on_dropfile will be removed in next two releases.')
+    def on_dropfile(self, filename):
         pass
 
+    def on_drop_text(self, text):
+        '''Event called when a text is dropped on the application.
+
+        .. note::
+            This event works with sdl2 window provider on x11 window.
+
+        .. versionadded:: 2.1.0
+        '''
```

Comment:
I'd add a `pass` here and below, although technically we don't need it.

Reply:
Done.